.github/workflows/test.yml

```diff
@@ -54,6 +54,11 @@ jobs:
     name: Build-master
     runs-on: ubuntu-latest
     steps:
+    # --squash is an experimental docker feature
+    - name: Enable Docker experimental features
+      run: |
+        echo $'{"experimental": true}' | sudo tee /etc/docker/daemon.json
```

Comment:
NIT: Is the '$' needed here? I'm not used to seeing that. what is the diff from just doing `echo '{"experimental": true}' | sudo tee /etc/docker/daemon.json`

Reply:
There's no need for the `$` here indeed.  There's no formatting inside the string.  I'll remove it.